pure.zsh

```diff
@@ -430,7 +432,13 @@ prompt_pure_setup() {
 	export PROMPT_EOL_MARK=''
 
 	# disallow python virtualenvs from updating the prompt
-	export VIRTUAL_ENV_DISABLE_PROMPT=1
+	if [ -z "${VIRTUAL_ENV_DISABLE_PROMPT-}" ]; then
+		# it was not set. set it to only have pure modify the shell.
+		export VIRTUAL_ENV_DISABLE_PROMPT=1
+	else
+		# it was already set. also disable pure display.
+		export PURE_VIRTUAL_ENV_DISABLE_PROMPT=1
```

Comment:
I would like to avoid the extra global variable, how about we set `VIRTUAL_ENV_DISABLE_PROMPT=2` instead of introducing a new variable? It's a bit more hidden but should serve the same purpose.